compiler/include/resolution.h

```diff
@@ -146,7 +146,8 @@ void wrapAndCleanUpActuals(ResolutionCandidate* best, CallInfo& info,
 typedef enum {
   FIND_EITHER = 0,
   FIND_REF,
-  FIND_NOT_REF
+  FIND_CONST_REF,
+  FIND_NOT_REF_OR_CONST_REF,
```

Comment:
It's ``!(ref || const_ref)``. But FIND_VALUE isn't good because there is also TYPE/PARAM ret tags. I'm happy to change the name if you have a suggestion.